roles/ceph-osd/tasks/get_osd_paths_dmcrypt.yml

```diff
@@ -0,0 +1,32 @@
+---
+  - name: get lockbox path for osd devices
+    shell: "df | grep {{ item }} | awk '{print $6}'"
+    with_items: "{{ devices }}"
+    changed_when: false
+    failed_when: false
+    check_mode: no
+    register: osd_path_lockbox
+
+  - name: get osd uuid from lockbox
+    shell: "cat {{ item.stdout }}/osd-uuid"
+    register: osd_uuids
+    changed_when: false
+    failed_when: false
+    check_mode: no
+    with_items: "{{ osd_path_lockbox.results }}"
+
+  - name: get osd dmcrypt device from uuid
+    shell: "readlink -f /dev/mapper/{{ item.stdout }}"
+    register: dmcrypt_devices
+    changed_when: false
+    failed_when: false
+    check_mode: no
+    with_items: "{{ osd_uuids.results }}"
+
+  - name: get osd path for dmcrypt devices
+    shell: "df | grep {{ item.stdout }} | awk '{print $6}'"
```

Comment:
ditto

Reply:
I can't. `command` does not support pipe.

Reply:
ditto for awk instead of grep+awk